src/test/java/eu/yals/test/app/TechPartsTest.java

```diff
@@ -19,121 +19,193 @@
+        assertEquals(500, result.getStatus());
 
-        TestUtils.assertResponseBodyNotEmpty(response);
-        TestUtils.assertContentType(MimeType.APPLICATION_JSON, response);
+        TestUtils.assertResponseBodyNotEmpty(result);
+        TestUtils.assertContentType(MimeType.APPLICATION_JSON, result);
     }
 
     @Test
-    public void onApiRequestWithoutHeadersAppReturnJsonWhenFailed() throws Exception {
-        HttpResponse<String> response = Unirest.get(TEST_URL + Endpoint.FAIL_API_ENDPOINT)
-                .asString();
+    public void onApiRequestWithNotJsonInAcceptHeaderAppReturns406WhenFailed() {
+        HttpRequest request = Unirest.get(TEST_URL + Endpoint.ForTests.FAIL_API_ENDPOINT)
+                .header(Header.ACCEPT, MimeType.APPLICATION_XML);
+        HttpResponse<String> result = request.asString();
 
-        assertEquals(500, response.getStatus());
+        logRequestAndResponse(request, result, TAG);
 
-        TestUtils.assertResponseBodyNotEmpty(response);
-        TestUtils.assertContentType(MimeType.APPLICATION_JSON, response);
+        assertEquals(406, result.getStatus());
     }
 
     @Test
-    public void onApiRequestWithNotJsonInAcceptHeaderAppReturns406WhenFailed() throws Exception {
-        HttpResponse<String> response = Unirest.get(TEST_URL + Endpoint.FAIL_API_ENDPOINT)
-                .header(Header.ACCEPT, MimeType.APPLICATION_XML)
-                .asString();
+    public void robotsTxtIsPresentAndText() {
+        HttpRequest request = Unirest.get(TEST_URL + Endpoint.Static.ROBOTS_TXT);
+        HttpResponse<String> result = request.asString();
+
+        logRequestAndResponse(request, result, TAG);
+
+        log.debug("Response: {}", result);
+        if (result == null) return;
+
+        Assert.assertEquals(200, result.getStatus());
+
+        String body = result.getBody();
+        Assert.assertTrue("robots.txt is empty", StringUtils.isNotBlank(body));
+        TestUtils.assertContentType(MimeType.TEXT_PLAIN, result);
+    }
+
+    @Test
+    public void humansTxtIsPresentAndText() {
+        HttpRequest request = Unirest.get(TEST_URL + Endpoint.Static.HUMANS_TXT);
+        HttpResponse<String> result = request.asString();
+
+        logRequestAndResponse(request, result, TAG);
+        log.debug("Response: {}", result);
+        if (result == null) return;
+
+        assertEquals(200, result.getStatus());
+
+        String body = result.getBody();
+        Assert.assertTrue("humans.txt is empty", StringUtils.isNotBlank(body));
+        TestUtils.assertContentType(MimeType.TEXT_PLAIN, result);
+    }
+
+    @Test
+    public void faviconIsPresentAndIcon() {
+        HttpRequest request = Unirest.get(TEST_URL + Endpoint.Static.FAVICON_ICO);
+        HttpResponse<String> result = request.asString();
+
+        logRequestAndResponse(request, result, TAG);
+        log.debug("Response: {}", result);
+        if (result == null) return;
+
+        Assert.assertEquals(200, result.getStatus());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)